src/main/java/me/coley/recaf/workspace/JarResource.java

```diff
@@ -29,6 +29,9 @@ public JarResource(File file) throws IOException {
 	@Override
 	protected Map<String, byte[]> loadClasses() throws IOException {
 		// iterate jar entries
+		ByteArrayOutputStream out = new ByteArrayOutputStream();
+		byte[] buffer = new byte[8192];
```

Comment:
For cases like this, how is the size of the buffer determined? I'm still curious as to why this is favorable over a one liner like `IOUtils.toByteArray(stream)`

Reply:
Answered in Discord.